pkgs/applications/window-managers/hyprwm/hyprland/plugins.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, callPackage
+, pkg-config
+, gcc13Stdenv
+, hyprland
+}:
+let
+  mkHyprlandPlugin =
+    a@{ pluginName, ... }:
+    gcc13Stdenv.mkDerivation (a // {
```

Comment:
```suggestion
    args@{ pluginName, ... }:
    gcc13Stdenv.mkDerivation (args // {
```
thats the standard reserved variable for those kinds of things

Reply:
fixed